From: SeongJae Park <sj@kernel.org>
Date: Mon, 1 Jul 2024 15:52:17 -0700
Subject: [PATCH] MAINTAINERS/MEMORY MANAGEMENT: add document files for mm

Memory managment subsystem documents ('admin-guide/mm/' and 'mm/' under
'Documentation/') are not marked as managed under memory management
subsystem.  This makes 'get_maintainer.pl' for changes to the documents
sub-optimal.  Mark the documents as part of mm subsystem on MAINTAINERS
file.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 MAINTAINERS | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index 8a18c2e135cd..2a2e92129646 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -14794,6 +14794,8 @@ S:	Maintained
 W:	http://www.linux-mm.org
 T:	git git://git.kernel.org/pub/scm/linux/kernel/git/akpm/mm
 T:	quilt git://git.kernel.org/pub/scm/linux/kernel/git/akpm/25-new
+F:	Documentation/admin-guide/mm/
+F:	Documentation/mm/
 F:	include/linux/gfp.h
 F:	include/linux/gfp_types.h
 F:	include/linux/memfd.h
-- 
2.39.5

